tests/fixups/TraceFixup/CreateProcessFixup.cpp

```diff
@@ -81,13 +120,53 @@ BOOL __stdcall CreateProcessAsUserFixup(
     _In_ startupinfo_t<CharT> startupInfo,
     _Out_ LPPROCESS_INFORMATION processInformation)
 {
-    auto entry = LogFunctionEntry();
+	LARGE_INTEGER TickStart, TickEnd;
+	QueryPerformanceCounter(&TickStart);    		auto entry = LogFunctionEntry();

```

Comment:
>    		 [](start = 37, length = 6)

formatting

Reply:
Addressed in commit 727a9e9856d361f686523f2ea2907876c3ef9b23 on 11/28